lib/social_view/element/friends_list.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+import 'package:image_stack/image_stack.dart';
+import 'package:lab_2_try/social_view/element/utils.dart';
+
+import '../../object/friends_obj.dart';
+
+class OnlineFriends extends StatelessWidget {
+  const OnlineFriends({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var phoneSize = MediaQuery.of(context).size;
+    return Container(
+        width: phoneSize.width,
+        margin: const EdgeInsets.only(left: 10, top: 5),
+        child: Align(
+            alignment: Alignment.centerLeft,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: const [
+                StyledText("Online", 25.0, Color(0xFFFFFFFF), FontWeight.w400),
```

Comment:
0xfff.. is just white. It's better to define some object with common colors (`class Theme { Color textColor = Colors.white; }`) and use them everywhere